data/korath ships.txt

```diff
@@ -25,6 +25,7 @@ ship "Korath Raider"
 		"outfit space" 721
```

Comment:
You should also adjust the outfit space, perhaps by half (if you go down to 1 integrated scoop) or in full if you keep the new scoop at 2.

Reply:
How about -6 outfits, +6 mass? This would result in free space going to 20/715, enough to fit two additional fuel pods or a hyperdrive alongside a jump drive (or vice versa).